tests/test_image_model_wrapper.py

```diff
@@ -43,6 +59,76 @@ def test_wrap_fastai_image_classification_model(self):
         wrapped_model = wrap_model(model, data, ModelTask.IMAGE_CLASSIFICATION)
         validate_wrapped_classification_model(wrapped_model, data)
 
+    @pytest.mark.parametrize("model_name", [ModelNames.SERESNEXT])
+    @pytest.mark.parametrize("multilabel", [False])
+    def test_wrap_automl_image_classification_model(self, model_name, multilabel):
+        data = load_fridge_dataset()
+
+        with tempfile.TemporaryDirectory() as tmp_output_dir:
+
+            task_type = shared_constants.Tasks.IMAGE_CLASSIFICATION
+            number_of_classes = 10
+            model_wrapper = ModelFactory().get_model_wrapper(model_name,
```

Comment:
I believe we need to add automl to test dependencies in order to run this test.  This could be difficult as it will bring a ton of dependencies to the tests and I'm not sure if there will be any conflicts/issues with installing all of the automl dependencies.

Reply:
thanks for helping with new build for automl Ilya!